### ISSUE TITLE
CPU Feature Detector

### ISSUE THREAD

**Issue by subh-kundu** [Mysticial/FeatureDetector#13]
Nice, please help me out how to work c# project please convert.

**Comment by Anime4000** [comment]
What I do, `Process RedirectStandardOutput` and use `RegEx` to capture strings, not so efficient as Native Hooking.
If this have `json` output, make code a little easier,

I hope it can run in batch/bash script and have return code:
```
FeatureDetector.exe --detect-avx
IF %ERRORLEVEL% NEQ 1 ( 
    echo This computer do not have AVX Instruction Set
    goto FAILED
)
```

**Comment by tifish** [comment]
> What I do, `Process RedirectStandardOutput` and use `RegEx` to capture strings, not so efficient as Native Hooking. If this have `json` output, make code a little easier,
> 
> I hope it can run in batch/bash script and have return code:
> 
> ```
> FeatureDetector.exe --detect-avx
> IF %ERRORLEVEL% NEQ 1 ( 
>     echo This computer do not have AVX Instruction Set
>     goto FAILED
> )
> ```

Agree. I add simple command line arguments and create a pull request. Binary available in my fork. Thanks @Mysticial to share the code.
